docs/reference/troubleshooting/disk/fix-master-node-out-of-disk.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[fix-master-node-out-of-disk]]
+== Fix master nodes out of disk
+
+Elasticsearch is using master nodes to coordinate the cluster. If one or more of these nodes are running
+out of space, you need to ensure that they have enough disk space to function. If the health disk indicator
```

Comment:
I think we should link to the health disk indicator docs here

Reply:
Added the health api reference doc.